src/sentry/api/endpoints/external_team.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+from django.db import IntegrityError
+
+from rest_framework import serializers, status
+from rest_framework.response import Response
+from sentry.api.serializers.rest_framework.base import CamelSnakeModelSerializer
+
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import ExternalTeam, EXTERNAL_PROVIDERS
+
+logger = logging.getLogger(__name__)
+
+
+class ExternalTeamSerializer(CamelSnakeModelSerializer):
+    team_id = serializers.IntegerField(required=True)
+    external_id = serializers.CharField(required=True)
```

Comment:
If this is an ID why is it a `CharField`? What does one look like? I'd assume it's an integer.

Reply:
@ceorourke `external_id` is set as a TextField in the ExternalTeam model and it maps to the Github/Gitlab team name. That's why we set it to CharField. https://github.com/getsentry/sentry/blob/master/src/sentry/models/integration.py#L87-L88